### ISSUE TITLE
Data Grid does not support some locale for DecimalPipe

### ISSUE THREAD

**Issue by Rado-1** [ng-matero/extensions#81]
If I used 'sk', 'cs', or 'de-*' locale for my application, or for particular module were the Data Grid was defined or for particular column by means of the MtxGridColumnTypeParameter.locale, the rendering throws an exception:
```
core.mjs:6469 ERROR Error: InvalidPipeArgument: 'Missing locale data for the locale "sk".' for pipe 'DecimalPipe'
    at invalidPipeArgumentError (common.mjs:4041:12)
    at DecimalPipe.transform (common.mjs:4801:19)
    at pureFunction3Internal (core.mjs:22149:1)
    at Module.ɵɵpipeBind3 (core.mjs:22333:1)
    at MtxGridCellComponent_ng_template_3_ng_container_6_Template (mtxGrid.mjs:255:5255)
    at executeTemplate (core.mjs:9603:1)
    at refreshView (core.mjs:9469:1)
    at refreshEmbeddedViews (core.mjs:10594:1)
    at refreshView (core.mjs:9493:1)
    at refreshEmbeddedViews (core.mjs:10594:1)
```
The component works fine for 'en-*' locales. It would be nice to support all locales which Angular does.

**Comment by nzbin** [comment]
Do you set other locales? Please check the docs  https://angular.io/api/core/LOCALE_ID

**Comment by Rado-1** [comment]
At the global level in `app.module.ts` I have:
```
providers: [
    { provide: LOCALE_ID, useValue: 'sk' },
    { provide: MAT_DATE_LOCALE, useValue: 'sk' },
  ],
```
This locale can be overridden in a lazy-loaded module which uses Data Grid. If I use, for instance:
```
 providers: [{ provide: LOCALE_ID, useValue: 'sk' }],
```
the error occurs. If I replace 'sk' by 'en-US' everything works fine.
The 3rd level of overriding locale is in MtxGridColumnTypeParameter. If I specify, e.g.:
```
typeParameter: {
        digitsInfo: '1.2-2',
        locale: 'sk',
      },
```
the error occurs.

I played with various variants of setting locale, e.g., used the same or different locales at different levels, omitted some levels of setting locales, etc.. Each time the Data Grid was supposed to use 'sk', the error occurred.

Note: I played primarily with 'sk' but the same situation is also for 'cs', 'de-*', etc.
